cdc/kv/metrics.go

```diff
@@ -16,12 +16,19 @@ package kv
 import "github.com/prometheus/client_golang/prometheus"
 
 var (
-	regionSplitCounter = prometheus.NewCounter(
+	eventFeedErrorCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: "ticdc",
 			Subsystem: "kvclient",
-			Name:      "region_split_count",
-			Help:      "The number of region split events since start.",
+			Name:      "event_feed_error_count",
+			Help:      "The number of error return by tikv",
+		}, []string{"type"})
+	eventFeedGauge = prometheus.NewGauge(
```

Comment:
Do you need `captureID` as a label?

Reply:
just display by instance, captureID is not readable and will not change, so i think no need to add by every metrics